packages/web3-providers/src/zerion/index.ts

```diff
@@ -170,3 +216,72 @@ export class ZerionAPI
         return createPageable(transactions, createIndicator(indicator))
     }
 }
+
+export class ZerionNonFungibleTokenAPI implements NonFungibleTokenAPI.Provider<ChainId, SchemaType> {
+    createNonFungibleTokenPermalink(address?: string, tokenId?: string) {
+        if (!address || !tokenId) return
+        return ZERION_NFT_DETAIL_URL + `${address}:${tokenId}`
```

Comment:
The NFT details link could vary for different EVM chains.

Reply:
Here is just the link to zerion's detail page. And zerion currently only supports Ethereum